CIP-1694/README.fr.md

```diff
@@ -0,0 +1,1186 @@
+---
+CIP: 1694
+Source: https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md
+Title: Un premier pas vers une gouvernance décentralisée on-chain
+Revision: b3e3508a
```

Comment:
```suggestion
Revision: b3e3508
```

Reply:
We could ask it in #542 😅 and we briefly mentioned & ruled out this idea in yesterday's CIP meeting... let's continue to talk about this prospect over there 🙏

Reply:
oops, you're correct, wrong place to comment that - I will move over to [#542](https://github.com/cardano-foundation/CIPs/pull/542).